src/Pos/Txp/Toil/Logic.hs

```diff
@@ -164,3 +172,39 @@ withTxId aux@(tx, _, _) = (hash tx, aux)
 
 applyTxToUtxo' :: MonadUtxo m => (TxId, TxAux) -> m ()
 applyTxToUtxo' (i, (t, _, d)) = Utxo.applyTxToUtxo (WithHash t i) d
+
+#ifdef WITH_EXPLORER
+modifyAddrHistory
```

Comment:
These functions should be moved to `cardano-explorer` and will be moved as a result of CSL-889. I don't think it makes sense to care about them now.